packages/key-did-resolver/src/__tests__/secp384r1.test.ts

```diff
@@ -0,0 +1,292 @@
+// Brent Shambaugh <brent.shambaugh@gmail.com>. 2021.
+
+import varint from "varint"
+import multibase from "multibase"
```

Comment:
the `multibase` package has been deprecated, could you update to use `multiformats`?
See https://github.com/ceramicnetwork/js-ceramic/pull/1862 for inspiration.

Reply:
Also, can you target the `release/2.x-progress` branch?